scanner/image/db/konflux.Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM registry.redhat.io/rhel8/postgresql-15:latest
+
+LABEL \
+    com.redhat.component="rhacs-scanner-v4-db-container" \
+    com.redhat.license_terms="https://www.redhat.com/agreements" \
+    description="Scanner v4 Database Image for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.description="Scanner v4 Database Image for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.display-name="scanner-v4-db" \
+    io.openshift.tags="rhacs,scanner-db,stackrox" \
+    maintainer="Red Hat, Inc." \
+    name="rhacs-scanner-v4-db-rhel8" \
+    source-location="https://github.com/stackrox/stackrox" \
+    summary="Scanner v4 DB for RHACS" \
+    url="https://catalog.redhat.com/software/container-stacks/detail/60eefc88ee05ae7c5b8f041c" \
+    # We must set version label to prevent inheriting value set in the base stage.
+    # TODO(ROX-20236): configure injection of dynamic version value when it becomes possible.
+    version="0.0.1-todo"
+
+USER root
+
+COPY --chown=postgres:postgres \
+     scanner/image/db/scripts/docker-entrypoint.sh \
+     scanner/image/db/scripts/init-entrypoint.sh \
+     /usr/local/bin/
+
+COPY db-init.dump.zst /db-init.dump.zst
```

Comment:
This is another case when the upstream has a blob
https://github.com/stackrox/stackrox/blob/07c64d0d43b4f37f92fbc6cc147f68fab680ab73/scanner/image/db/Dockerfile#L28-L28

but the downstream doesn't (not even `pre-build-script` for fetching).
https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/blob/rhacs-1.0-rhel-8/distgit/containers/rhacs-scanner-v4-db/Dockerfile.in?ref_type=heads

Should this be added to the downstream?

https://redhat-internal.slack.com/archives/C033SKHG9CZ/p1710776736986199

Reply:
Based on that conversation, I've removed the init bundle from the konflux dockerfile and tekton pipelines: 7c591b8